makefile.shade

```
@@ -926,6 +934,20 @@ functions @{
         }
     }
 
+    public bool LinuxDistro(string distro)
+    {
+        try
+        {
+            string result;
+            ExecuteAndRedirectOutput("cat", "/etc/os-release", out result);
+            return result.Contains(distro);
+        }
+        catch
+        {
+            return false;
```

Maybe just let it fail? It would have saved us some time yesterday ;)